consensus/src/experimental/persisting_phase.rs

```diff
@@ -67,9 +68,14 @@ impl StatelessPipeline for PersistingPhase {
             commit_ledger_info,
             callback,
         } = req;
+        let log = commit_ledger_info.commit_info().clone();
+        debug!("Receive Persist request for {}", log);
 
-        self.persisting_handle
+        let result = self
+            .persisting_handle
             .commit(&blocks, commit_ledger_info, callback)
-            .await
+            .await;
+        debug!("Finish Persist request for {}", log,);
```

Comment:
nit: remove the extra comma?

Reply:
didn't mean to commit it, just for smart people to reproduce